client/src/components/widgets/MessageDisplay.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <div
+    class="alert m-3"
+    :class="alertType">
+    <font-awesome-icon class="icon" :icon="['fas', iconType]"/>
+    <slot name="message" />
+  </div>
+</template>
+
+<script lang="ts">
+  import Vue from 'vue';
+  import Component from 'vue-class-component';
+  import { Prop } from 'vue-property-decorator';
+
+  @Component
+  export default class Counters extends Vue {
```

Comment:
class name should prob be `MessageDisplay`

Reply:
Good catch!

Reply:
Thanks, good catch.

Reply:
Fixed.